integration/util/scenario/starport.js

```diff
@@ -21,8 +21,7 @@ class Starport {
   }
 
   chainAddressStr() {
-    // xxx todo:wn parameterize by chain
-    return `ETH:${this.ethAddress()}`;
+    return `${this.chain.name.toUpperCase()}:${this.ethAddress()}`;
```

Comment:
Still not generalized though, right?

Reply:
Yeah starport.js is quite eth specific right now, that refactor will most likely come with flow